### ISSUE TITLE
Autopkg 2 Issues - GoogleTalkPlugin - MunkiPkginfoReceiptsEditor

### ISSUE THREAD

**Issue by TSPARR** [autopkg/keeleysam-recipes#112]
I receive the following error in Autopkg 2. 
```
WARNING: /Users/munki/Library/AutoPkg/RecipeRepos/com.github.autopkg.keeleysam-recipes/GoogleTalkPlugin/MunkiPkginfoReceiptsEditor.py: cannot import name 'readPlist' from 'FoundationPlist' (/Library/AutoPkg/FoundationPlist/__init__.py)
Traceback (most recent call last):
  File "/Library/AutoPkg/autopkglib/__init__.py", line 820, in get_processor
    _tmp = imp.load_source(processor_name, processor_filename)
  File "/Library/AutoPkg/Python3/Python.framework/Versions/Current/lib/python3.7/imp.py", line 171, in load_source
    module = _load(spec)
  File "<frozen importlib._bootstrap>", line 696, in _load
  File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/Users/munki/Library/AutoPkg/RecipeRepos/com.github.autopkg.keeleysam-recipes/GoogleTalkPlugin/MunkiPkginfoReceiptsEditor.py", line 6, in <module>
    from FoundationPlist import readPlist, writePlist
ImportError: cannot import name 'readPlist' from 'FoundationPlist' (/Library/AutoPkg/FoundationPlist/__init__.py)
  File "/Library/AutoPkg/autopkglib/__init__.py", line 820, in get_processor
    _tmp = imp.load_source(processor_name, processor_filename)
Failed.
```

**Comment by n8felton** [comment]
I've been finding myself using this processor a bit more as I like the ability to mark receipts as optional for things like MAU in Microsoft installers after using installer_choices to prevent it from being installed.

I'd like to recommend this be moved to a `SharedProcessors` location (and out of GoogleTalkPlugin).

**Comment by TSPARR** [comment]
This appears to have been resolved by d097018.
